WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -462,10 +470,8 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 if (resultCode == Activity.RESULT_OK) {
                     fetchMedia(UCrop.getOutput(data));
                 } else if (resultCode == UCrop.RESULT_ERROR) {
-                    Toast.makeText(getActivity(), getString(R.string.error_cropping_image), Toast.LENGTH_SHORT).show();
-
-                    final Throwable cropError = UCrop.getError(data);
-                    AppLog.e(AppLog.T.MAIN, "Image cropping failed!", cropError);
+                    AppLog.e(AppLog.T.MAIN, "Image cropping failed!", UCrop.getError(data));
```

Comment:
AS is complaining that `UCrop.getError(data)` can be `null`, but I don't see a good way around that.